doc/changelog/master.md

```diff
@@ -293,11 +272,7 @@ https://github.com/azerothcore/mod-server-auto-shutdown
 - Just rename all hooks from `CanExitJoinMessageArenaQueue` and `CanSendMessageArenaQueue`, to `OnBeforeSendExitMessageArenaQueue`
 - Just rename all hooks from `CanSendJoinMessageArenaQueue` and `OnBeforeSendJoinMessageArenaQueue`
 
-## 4.0.0-dev.3 | Commit: [c35dde6fae732269357b78fb796fba21956b83fc
-](https://github.com/azerothcore/azerothcore-wotlk/commit/c35dde6fae732269357b78fb796fba21956b83fc
-
-
-Changelog for commit "[refactor(Collision): Update some methods to UpperCamelCase](https://github.com/azerothcore/azerothcore-wotlk/commit/b84f9b8a4b334632cb37dcebbb2dd4e087f65610)"
+## 4.0.0-dev.3 | Commit: [c35dde6fae732269357b78fb796fba21956b83fc](https://github.com/azerothcore/azerothcore-wotlk/commit/c35dde6fae732269357b78fb796fba21956b83fc
```

Comment:
you need to close the parenthesis `)`, otherwise the link is not working

Reply:
(the error is present in the other hash links)

Reply:
ah missed that. However, then there are the 2 issues. as the closing square bracket gets put on a newline for no reason